docs/content/usage/reports/mitre.md

```diff
@@ -18,13 +18,14 @@ Watch an overview video:
 
 The MITRE ATT&CK report is centred around the ATT&CK matrix:
 
-![MITRE Report](/images/usage/reports/mitre-report-0.jpg "MITRE Report")
+![MITRE Report](/images/usage/reports/mitre-report-0.png "MITRE Report")
 
 The Monkey rates your network on the attack techniques it attempted. For each technique, you can get
 
 - {{< label danger Red >}}: The Monkey **successfully used** the technique in the simulation. That means your network is vulnerable to this technique being employed.
 - {{< label warning Yellow >}}: The Monkey **tried to use** the technique, but didn’t manage to. That means your network isn’t vulnerable to the way Monkey employs this technique.
-- {{< label other Grey >}}: The Monkey **didn't try** the technique this time. Perhaps it wasn't relevant to this network or wasn't configured.
+- {{< label unused "Dark Gray" >}}: The Monkey **didn't try** the technique this time. Perhaps it wasn't relevant to this network.
+- {{< label disabled "Light Gray" >}}: The Monkey **didn't try** the technique this time since it wasn't configured.
```

Comment:
```suggestion
- {{< label unused "Dark Gray" >}}: The Monkey **didn't try** the technique. Perhaps it wasn't relevant to this network.
- {{< label disabled "Light Gray" >}}: The Monkey **didn't try** the technique since it wasn't configured.
```

Reply:
Done